static/js/ducks/galaxies.js

```diff
@@ -20,8 +20,12 @@ export function fetchGalaxies(filterParams = {}) {
 
 export function fetchGcnEventGalaxies(dateobs = null, filterParams = {}) {
   filterParams.localizationDateobs = dateobs;
-  filterParams.localizationCumprob = 0.95;
-  filterParams.includeGeojson = true;
+  filterParams.includeGeoJSON = true;
+
+  if (!Object.keys(filterParams).includes("localizationCumprob")) {
+    filterParams.localizationCumprob = 0.95;
```

Comment:
I'm not sure why the default is added here and not on the handler (the python side)?

Reply:
It's added both places... do you want it removed?

Reply:
I think that the duck should not be setting defaults, if those defaults are set lower down in the back-end. If the front-end renders correctly because you explicitly call the option you want, then we should leave the behavior of that call to whatever default is on the back-end.